packages/grid_client/scripts/connection.ts

```diff
@@ -0,0 +1,90 @@
+/* 
+  Use this script to test grid connection:
+  This script creates 2 instances of the grid client, then after 10 minutes, it performs a query on both instances.
+
+  Steps:
+    1 - Connect to the grid
+    2 - Close the internet connection
+    3 - After 9 minutes, open the internet connection again
+  The grid should automatically reconnect and request the twin ID.
+*/
+
+import { GridClient, NetworkEnv } from "../src";
+
+// Replace with actual mnemonics
+const clientMnemonic1 = "your_mnemonic_here";
+const clientMnemonic2 = "your_mnemonic_here";
+const storeSecret = "store_secret";
+
+// Choose the desired network (main, test, qa, dev)
+const NETWORK = NetworkEnv.dev;
+
+// Function to log the current minute along with the client's name
+function logCurrentMinute(clientName) {
+  const now = new Date();
+  console.log(`${clientName} running in minute ${now.getMinutes()}`);
+}
+
+// Client 1 logic
+async function client1() {
+  const options = {
+    network: NETWORK,
+    mnemonic: clientMnemonic1,
+    storeSecret: storeSecret,
+  };
+  try {
+    const gridClient = new GridClient(options);
+    await gridClient.connect();
+
+    // Log the current minute every minute
+    setInterval(() => {
+      logCurrentMinute("Client 1");
+    }, 60000); // Log every minute (60000 milliseconds)
```

Comment:
do we need to do that? it says nothing imo. we need to know start of the connection only I think